### PR TITLE
Added category: Music

### PR DESCRIPTION
**Pull request by elianiva**

### CODE REVIEW

File: qual_id/pattern.py [hunk 1]
```diff
@@ -19,6 +20,7 @@ class Pattern:
       "game":Game(),
       "geography":Geography(),
       "instrument": Instrument()
```
Comment (gabrielbarker) [inline qual_id/pattern.py:22]:
```suggestion
      "instrument": Instrument(),
```
This is making the tests fail, I'll fix it when I resolve the conflicts. They're just due to other PRs getting merged 🙂

**Review by gabrielbarker — APPROVED** [review]
I'll make that tiny change and fix the conflicts and then it's ready to merge

**Comment by gabrielbarker** [comment]
Thanks a lot for the contribution @elianiva ! Nice work 🎉

**Comment by elianiva** [comment]
Oops, somehow I forgot the comma there. Thanks for fixing it :D

**Pull request merged**